src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,76 @@
+const Task = require('../../models/task/task');
```

Comment:
why do you need a "task" folder, and then another "task" file. Delete the "task" folder